### PR TITLE
MBLD scrambling guidelines

### PR DESCRIPTION
**Pull request by Rouxles**

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [ADDITION] For Multi-Blind Solving, the scrambler should verify that the puzzles are scrambled correctly and that there are no duplicate scrambles.
```
Comment (Nanush7) [inline wca-guidelines.md]:
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should pay special attention to duplicate scrambles.
```

I think it's better to remove that part, so we don't give the idea that checking the scrambles is a "should". 4g already states that they must do it.

File: wca-guidelines.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should perform an extra check after scrambling to verify there are no duplicate scrambles.
```
Comment (CarterKoala) [inline wca-guidelines.md]:
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler should verify there are no duplicate scrambles.
```

**Review by CarterKoala — COMMENTED** [review]
I don't like using check as a noun here, I think this is better.

File: wca-guidelines.md (hunk 2 above)
Comment (Nanush7) [reply wca-guidelines.md]: LGTM.

**Comment by Rouxles** [comment]
This one will need a review from somebody

File: wca-guidelines.md [hunk 3]
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler should verify there are no duplicate scrambles.
```
Comment (Nevseros) [inline wca-guidelines.md]:
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.
```

**Review by Nevseros — APPROVED** [review]
LGTM!

File: wca-guidelines.md [hunk 4]
```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [RECOMMENDATION] For Multi-Blind Solving, after scrambling all the cubes, the scrambler must verify there are no duplicate scrambles.
```
Comment (Samuel-Baird) [inline wca-guidelines.md:96]: it seems a bit odd to list a "must regulation" as a "recommendation". Perhaps "clarification" would be better suited?

Comment (CarterKoala) [reply wca-guidelines.md:96]: Indeed, this should not be a recommendation. It will be fixed :)

**Pull request merged**